support-workers/src/main/scala/com/gu/support/workers/lambdas/PaymentMethodExtensions.scala

```diff
@@ -1,13 +1,19 @@
 package com.gu.support.workers.lambdas
 
-import com.gu.support.workers.{CreditCardReferenceTransaction, DirectDebitPaymentMethod, PayPalReferenceTransaction, PaymentMethod}
+import com.gu.support.workers.{
```

Comment:
interesting way to write imports, is that normal?

Reply:
just it was complaining about the line length, but wasn't sure whether to a an `_` import. Thoughts?

Reply:
fine by me! just looked funny.  I'm not a big fan of _ imports anyway but intellij used to revert to them pretty quickly